### ISSUE TITLE
Birthday suggestion

### ISSUE THREAD

**Issue by AirFusion45** [AirFusion45/Power-Bot#23]
@deleted-role  Suggestion for @PowerBot : have a kind of calender for birthdays. You could add the persons name and the date with a command, and remove it too. The bot would then congratulate that person or announce it on the discord server - Alex from AllCraft

**Comment by AirFusion45** [comment]
* Find a way to trigger a event when `currentTime = <entered birthday time>`
* store the birthdays in a `birthdays.json`? and `if (birthday.one = date.now())`? then => `exec happy bday code`?

**Comment by AirFusion45** [comment]
unfortunately, this will be pretty hard to implement, I am not going to launch this on PoiwerBot, so it will only be a signature edition feature, and we need to keep track which server is the message being sent to,  who, when, in a separate database. I will put this on hold, and work on it if I really get support for it.

**Comment by ethamitc** [comment]
This could be done very easily using Enmap and storing the date as the key and the user ID & Guild ID as the value.

**Comment by AirFusion45** [comment]
yes @ethamitc I agree with you now. A function that runs every day to check if that day has a birthday in enmap. This is not hard to implement, you can implement after acquisition.
